src/NuGet.Clients/PackageManagement.PowerShellCmdlets/NuGetPowerShellBaseCommand.cs

```diff
@@ -193,6 +196,11 @@ protected IErrorHandler ErrorHandler
             get { return this; }
         }
 
+        /// <summary>
+        /// Determine if needs to log total time elapsed or not
+        /// </summary>
+        protected bool IsLoggingTimeEnabled { get; set; }
```

Comment:
This should be a virtual get only property, and then you never need to set the value. Make it `IsLoggingTimeDisabled` so its off by default and then you only need to return true in one implementation